rclcpp/include/rclcpp/create_timer.hpp

```diff
@@ -90,15 +90,17 @@ create_timer(
   rclcpp::Clock::SharedPtr clock,
   rclcpp::Duration period,
   CallbackT && callback,
-  rclcpp::CallbackGroup::SharedPtr group = nullptr)
+  rclcpp::CallbackGroup::SharedPtr group = nullptr,
+  bool start_canceled = false)
```

Comment:
how about the following? just a suggestion.

```suggestion
  bool auto_start = true)
```